RealmSwift/Aliases.swift

```diff
@@ -74,6 +74,38 @@ extension ObjectBase {
      transactions it will be called at some point in the future after the write
      transaction is committed.
 
+     If no key paths are given, the block will be executed on any insertion,
+     modification, or deletion for all object properties and nested linked
+     properties (up to a depth of 4). If a key path or key paths are provided,
```

Comment:
If we name exceptions here, should `LinkingObjects` be named too?

Reply:
Since LinkingObjects behavior is still in the air, I'll address once we're sure.